### PR TITLE
Trib 137

### PR DESCRIPTION
**Pull request by loretdemolas**

### CODE REVIEW

**Review by haxwell — CHANGES_REQUESTED** [review]

**Comment by haxwell** [comment]
Also, there are conflicts. Can you resolve those? Let me know if you need some advice.

File: src/test/java/com/savvato/tribeapp/services/NotificationServiceImplTest.java [hunk 1]
```diff
@@ -41,6 +50,37 @@ public NotificationService notificationService() {
 	@MockBean
 	NotificationRepository notificationRepository;
 
+
+	@Test
+	public void testCreateNotificationDTO() {
+		// Mock data
+		NotificationType mockType = new NotificationType();
```
Comment (haxwell) [inline src/test/java/com/savvato/tribeapp/services/NotificationServiceImplTest.java:57]: Use an actual NotificationType instance, like NotificationType.MEDIUM_PRIORITY, or whatever's defined.

File: src/main/java/com/savvato/tribeapp/services/NotificationServiceImpl.java [hunk 2]
```diff
@@ -78,4 +84,20 @@ public Notification createNotification(NotificationType type, Long userId, Strin
 
         return null;
     }
+    //tested
```
[the remark this answers is not in the document]
Comment (loretdemolas) [reply src/main/java/com/savvato/tribeapp/services/NotificationServiceImpl.java]: done.

File: src/test/java/com/savvato/tribeapp/services/NotificationServiceImplTest.java (hunk 1 above)
Comment (haxwell) [reply src/test/java/com/savvato/tribeapp/services/NotificationServiceImplTest.java]: It looks like you went the other direction, and removed the testing of NotificationType. I believe that still needs to be included in the test, DTO, and associated objects.

File: src/main/java/com/savvato/tribeapp/repositories/NotificationRepository.java [hunk 3]
```diff
@@ -2,8 +2,14 @@
...
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface NotificationRepository extends JpaRepository<Notification, Long> {
+
+    @Query("SELECT n FROM Notification n WHERE n.userId = :userId")
```
[the remark this answers is not in the document]
Comment (loretdemolas) [reply src/main/java/com/savvato/tribeapp/repositories/NotificationRepository.java]: Removed and works fine

**Pull request merged**